src/demux.js

```diff
@@ -295,6 +358,8 @@ class DemuxSink extends Writable {
         // Subsequent operations will be taken by services in the upper level, like upload to store and persist
         // metadata do a db.
         case 'close':
+            this.log.info('[Demux] sink closed');
```

Comment:
In case a client reconnect happens, the client won't sent the identity data again, which means the meta object will be missing the identity information used in the app.

Reply:
the startDate that's set in `_sinkCreate` will also not correspond with the actual start date of the session, won't this affect the rest of the application?